scintilla/lexers/LexUser.cxx

```diff
@@ -635,13 +635,13 @@ static inline void SubGroup(const char * s, vvstring & vec, bool group=false)
     for (unsigned int j=0; j<length+1; ++j)
         temp[j] = 0;
 
-    if (s[0] == '(' && s[1]  == '(')

```

Comment:
Are you sure that you want to modify core `scintilla` code? That doesn't seem like a good idea.

What's the reasoning for these changes? Would `scintilla` accept a Pull Request instead?